notebooks/README.md

```diff
@@ -1,24 +1,24 @@
-| [205-vision-background-removal](205-vision-background-removal/205-vision-background-removal.ipynb) <br><br> [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F205-vision-background-removal%2F205-vision-background-removal.ipynb) | <ul><li>Demonstrates background removal in images</li><li>The open source [U^2-Net](https://github.com/xuebinqin/U-2-Net) model is converted from PyTorch</li><li>Users can upload their own images to remove and replace the background with OpenVINO and OpenCV</li></ul> | <img src="https://user-images.githubusercontent.com/15709723/125184237-f4b6cd00-e1d0-11eb-8e3b-d92c9a728372.png"> |
-| [206-vision-paddlegan-anime](206-vision-paddlegan-anime/206-vision-paddlegan-anime.ipynb) <br><br> [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F206-vision-paddlegan-anime%2F206-vision-paddlegan-anime.ipynb) | <ul><li>Demonstrates using PaddlePaddle models with OpenVINO</li><li>The open source [AnimeGAN](https://github.com/PaddlePaddle/PaddleGAN) model is converted with paddle2onnx and then OpenVINO IR with Model Optimizer</li><li>Shows inference results on the AnimeGAN model</li><li>Benchmark app shows performance on CPU, iGPU and both devices together using [MULTI plugin](https://docs.openvinotoolkit.org/latest/openvino_docs_IE_DG_supported_plugins_MULTI.html)</li><li>Users can upload their own images to try out</li></ul> | <img src="https://user-images.githubusercontent.com/15709723/125184441-b4584e80-e1d2-11eb-8964-d8131cd97409.png"> |
+| [001-hello-world](001-hello-world/001-hello-world.ipynb) &nbsp; [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F001-hello-world%2F001-hello-world.ipynb) | <ul><li>A very basic introduction to OpenVINO's [Python API](https://docs.openvinotoolkit.org/latest/ie_python_api/annotated.html)</li><li> Shows how to do inference on a [mobilenetv3](https://docs.openvinotoolkit.org/latest/omz_models_model_mobilenet_v3_small_1_0_224_tf.html) image classification model</li></ul> | <img src="001-hello-world/data/classification.jpg" width=300><br>   |
+| [002-openvino-api](002-openvino-api/002-openvino-api.ipynb) [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F002-openvino-api%2F002-openvino-api.ipynb)  | OpenVINO API tutorial that covers the following: <ul><li>Load Inference Engine and Show Info</li><li>Loading a Model</li><ul><li>IR Model</li><li>ONNX Model</li></ul><li>Getting Information about a Model</li><ul><li>Model Inputs</li><li>Model Outputs</li></ul><li>Doing Inference on a Model</li><li>Reshaping and Resizing</li><ul><li>Change Image Size</li><li>Change Batch Size</li></ul></ul> | <img src="002-openvino-api/data/readme_image.jpg" ><br>  |
```

Comment:
Same thing for this.. make a githubusercontent URL: https://user-images.githubusercontent.com/15709723/127032986-32163c05-9837-4261-bcdf-42ad2a527922.jpg

![readme_image](https://user-images.githubusercontent.com/15709723/127032986-32163c05-9837-4261-bcdf-42ad2a527922.jpg)

Reply:
Done